test/json_schema/validator_test.rb

```diff
@@ -794,6 +798,19 @@
   end
 =end
 
+  it "validates custom formats" do
```

Comment:
For this one would you mind renaming to "validates custom formats unsuccessfully" and then add a second test case called "validates custom formats successfully" that asserts that it works as long as the value is indeed 42?

There should be some other examples of this pattern for other formats in the file (look for "validates uuid format ..." for example).